cdc/puller/multiplexing_puller.go

```diff
@@ -254,30 +322,38 @@ func (p *MultiplexingPuller) run(ctx context.Context, includeClient bool) error
 			zap.String("changefeed", p.changefeed.ID))
 	}()
 
-	g, ctx := errgroup.WithContext(ctx)
```

Comment:
I prefer `g`.

Reply:
The term "eg" is used elsewhere in the codebase, so I chose to maintain a consistent style here.

Reply:
`g` is also used elsewhere, it's not a big matter, just ignore it. looks good to me